docs/content/en/docs/reference/clusterspec/vsphere.md

```diff
@@ -255,16 +255,17 @@ More details on the driver are [here](https://docs.vmware.com/en/VMware-vSphere-
 > rely on the other resources. This should be done after setting `disableCSI` to `true` and running `upgrade cluster`.
 > 
 > These are the resources you would need to delete:
-> * vsphere-csi-controller-role (kind: ClusterRole)
-> * vsphere-csi-controller-binding (kind: ClusterRoleBinding)
-> * csi.vsphere.vmware.com (kind: CSIDriver)
+> * `vsphere-csi-controller-role` (kind: ClusterRole)
+> * `vsphere-csi-controller-binding` (kind: ClusterRoleBinding)
+> * `csi.vsphere.vmware.com` (kind: CSIDriver)
+> * `<cluster-name>-csi` (kind: ClusterResourceSet)
```

Comment:
This one is specifically in the `eksa-system` for the management cluster right? These other resources that are being deleted are on the target cluster itself. We need a separate point for that and run it by @chrisnegus